### PR TITLE
Dependency Updates

### PR DESCRIPTION
**Pull request by mhottinger1**
### Description:
Updated some dependencies
* Spring Boot: 1.5.21 to 1.5.22
* Commons DBCP -> Commons DBCP2

### Motivation:
Prepare scheduler code base for upgrade to Spring Boot 2.

### CODE REVIEW

File: scheduler/db/autoscaler_tables_postgres.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
+CREATE TABLE app_scaling_specific_date_schedule
```
Comment (qibobo) [inline scheduler/db/autoscaler_tables_postgres.sql:1]: what is this sql for?

Comment (mhottinger1) [reply scheduler/db/autoscaler_tables_postgres.sql:1]: Hey @qibobo it's a convenience sql script that I used to get `mvn test` to run successfully. There's a couple tests in TestDataDbUtil.java that require the tables/columns in that script to exist.

Comment (qibobo) [reply scheduler/db/autoscaler_tables_postgres.sql:1]: @mhottinger1 
We use [liquibase](https://github.com/cloudfoundry/app-autoscaler#initialize-the-database) to initialize the database. So I think the sql script is a duplication.

Comment (mhottinger1) [reply scheduler/db/autoscaler_tables_postgres.sql]: dupe sql script removed per feedback.

**Comment by cdlliuy** [comment]
@mhottinger1 ,  can you double check with your CLA status? 
It seems that the easyCLA tools is report wrong status. 
https://github.com/cloudfoundry/app-autoscaler/pull/565#issuecomment-622438493

**Comment by mhottinger1** [comment]
hey @cdlliuy, [per GitHub](https://help.github.com/en/github/using-git/changing-author-info) it appears the only way to remedy this situation is to re-write commit history. based on the results above might be best to reject this PR and i can submit a new one with a clean easyCLA check.

**Comment by mhottinger1** [comment]
Closing this pull request and will open a new one.

**Pull request closed without merging**